docs/guides/assets.md

```diff
@@ -242,9 +246,29 @@ Calling this API will set the asset base path for all Stencil components attache
 As a result, calling `setAssetPath` should not be done from within a component in order to prevent unwanted side effects
 when using a component.
 
-If the file calling `setAssetPath` is a module, it's recommended to use `import.meta.url`.
+If the file calling `setAssetPath` is a module and your compiler is able to resolve the file location at build time, based on where the application is being deployed, you can use `import.meta.url`. In most cases however we recommend to use:
+
+```ts
+import { setAssetPath } from '@stencil/core';
+setAssetPath(`${window.location.origin}/`);
+```
 
-Alternatively, one may use [`document.currentScript.src`](https://developer.mozilla.org/en-US/docs/Web/API/Document/currentScript)
-when working in the browser and not using modules or environment variables (e.g. `document.env.ASSET_PATH`) to set the
-asset base path. 
-This configuration depends on how your script is bundled, (or lack of bundling), and where your assets can be loaded from.
+Alternatively, one may use [`document.currentScript.src`](https://developer.mozilla.org/en-US/docs/Web/API/Document/currentScript) when working in the browser and not using modules or environment variables (e.g. `document.env.ASSET_PATH`) to set the
+asset base path. This configuration depends on how your script is bundled, (or lack of bundling), and where your assets can be loaded from.
+
+For example, in case you import a component directly via script tag, this would look like:
+
+```html
+<html>
+  <head>
+    <script src="https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic.js"></script>
+    <script type="module">
+      import { setAssetPath } from 'https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic.js';
```

Comment:
If I copy/paste this into a local file, I get "import not found: setAssetPath" - is this the bug you were referring to in Slack this morning/last night?
![Screenshot 2024-05-01 at 8 54 06 AM](https://github.com/ionic-team/stencil-site/assets/1930213/2c4d40d8-2b5e-4adf-a868-99ae8a0f79c0)

Reply:
Correct, I will try to address this in the docs until we have a fix for this.